apps/src/sites/code.org/pages/public/teacher-dashboard/sections.js

```diff
@@ -33,9 +31,6 @@ export function renderSectionsPage(data) {
   store.dispatch(setStudioUrl(data.studiourlprefix));
   store.dispatch(setValidLoginTypes(data.valid_login_types));
   store.dispatch(setValidGrades(data.valid_grades));
-  if (!experiments.isEnabled('sectionFocus')) {
-    store.dispatch(setValidAssignments(data.valid_courses, data.valid_scripts));
```

Comment:
Right you are. This should also allow us to remove a chunk of code from pegasus :) I'll do that as a follow on PR, since this one is green and we'd like to get this shipped.